src/compat/db_date.c

```diff
@@ -1390,15 +1396,20 @@ parse_mtime (const char *buf, int buf_len, unsigned int *mtime, bool * is_msec,
 	{
 	  for (c = *p; p < strend && char_isspace (c); c = *++p)
```

Comment:
Isn't it possible that at the end , the step `c = *++p` reaches outside the buffer? I also attempted to fix same issues in https://github.com/CUBRID/cubrid/pull/1562/files. However, I have some regressions I did not fix, because it was lower prority to me.